.stylelintrc.cjs

```diff
@@ -14,17 +14,31 @@ module.exports = {
         'unit-disallowed-list': [
           ['rem', 'em'],
         ],
+        // TODO: Enable once all deprecated CSS custom properties have been removed in breaking version.
+        //
+        // Only allow @kong/design-tokens CSS custom properties
+        // 'custom-property-pattern': [
+        //   "^(kui).+$",
+        //   {
+        //     message: "Expected custom property \"%s\" to be sourced from @kong/design-tokens with prefix '--kui-'",
+        //   }
+        // ],
```

Comment:
I'm leaving this disabled for now until after we remove utility css custom properties in the breaking release